lib/hooks/comment/index.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = settings => () => {
+  return Promise.resolve();
```

Comment:
Do you have a notion of what might appear in here?

I'd originally seen this and assumed it was a notify hook, which makes a lot of sense, but I can't imagine what standalone stuff should be in here.

Reply:
done